### ISSUE TITLE
What changes do i need to make inorder to train it to recoginize both alphabets and numbers?

### ISSUE THREAD

**Issue by arsalan993** [MaybeShewill-CV/CRNN_Tensorflow#31]
You previously told me to make changes in word dictionary. 
Can you tell me exactly where i need to make changes.

**Comment by TJCVRS** [comment]
@arsalan993 You need to adjust the char dict in data/char_dict folder to fit your own dataset

**Comment by arsalan993** [comment]
Yes i have seen the character map which you have uploaded but i am unable to understand it..  Can you please elaborate it a bit.. there are two file and i am unable to get whats in it.

**Comment by arsalan993** [comment]
If i am not wrong you have included the numeric character map as well
`"0": "97", "1": "98", "2": "99", "3": "100", "4": "101", "5": "102", "6": "103", "7": "104", "8": "105", "9": "106", "10": "107", "11": "108", "12": "109", "13": "110", "14": "111", "15": "112", "16": "113", "17": "114", "18": "115", "19": "116", "20": "117", "21": "118", "22": "119", "23": "120", "24": "121", "25": "122", "26": "48", "27": "49", "28": "50", "29": "51", "30": "52", "31": "53", "32": "54", "33": "55", "34": "56", "35": "57", "36": "32"`

but i found this on order map.json file 
but whats on the char_dict.json i am unable to understand it

**Comment by rkshuai** [comment]
The synt90 dataset has few numbers， like this

![wx20180130-144856](https://user-images.githubusercontent.com/11296462/35552084-4f72d2f4-05cd-11e8-8f30-6513743f6af9.jpg)

**Comment by TJCVRS** [comment]
@arsalan993 The synth90 dataset do have the number character, if what you want is only to recognize numbers you can just remove all the letter in the char_dict.txt the reconstruct the json file.

**Comment by arsalan993** [comment]
Actually i want to recognize both English alphabet characters and numeric characters as well.. 
But when i run your pretrained model..  it only recognizes the English alphabets not the numbers

**Comment by TJCVRS** [comment]
@arsalan993 you may training your own model to fit your dataset

**Comment by arsalan993** [comment]
Any update on pre-trained model on complete data set

**Comment by arsalan993** [comment]
i waiting for the model which is trained on complete synth90 dataset, and should be able to recognize both characters and numbers 0-9 ... 
I dont have enough resources to train a new model..  as u have said you will do it can you please do it in near time.

**Comment by TJCVRS** [comment]
@arsalan993 Sorry about that. I have not gotten enough time to do it and I will do it once I get enough time.

**Comment by mdbenito** [comment]
Note that recent changes allow for automatic charset creation from the labels in the dataset. Making this process easier. See PRs #125 and #145 (already merged into master)

**Comment by MaybeShewill-CV** [comment]
@mdbenito Thanks for your great work
